### ISSUE TITLE
adapt pth_06 bloom-json database schema changes to bloomfilter generation code

### ISSUE THREAD

**Issue by kortemik** [teragrep/pth_10#183]
**Description**
filtertype and bloomfilter tables have been created instead of multi-table layout. adapt to use these.

insert first all the json configured types to filtertype table
second insert filters as they are generated with a query that resembles following
```
insert into bloomfilter (type_id, filter) values ((select id from filtertype where expected_items=? and targetfpp=?), ?);
```

**Comment by kortemik** [comment]
see into https://github.com/teragrep/pth_06/issues/22 for the counterpart

**Comment by elliVM** [comment]
Added schema changes, fixed tests

**Comment by kortemik** [comment]
implemented in 4.15.0
